test/testutil/tests.c

```diff
@@ -293,7 +164,7 @@ DEFINE_COMPARISONS(char, char, "%c")
 DEFINE_COMPARISONS(unsigned char, uchar, "%u")
 DEFINE_COMPARISONS(long, long, "%ld")
 DEFINE_COMPARISONS(unsigned long, ulong, "%lu")
-DEFINE_COMPARISONS(size_t, size_t, "%" OSSLzu)
+DEFINE_COMPARISONS(size_t, size_t, "%zu")
```

Comment:
this doesn't need the same "not universally supported format" ifdef from the header file above?